### PR TITLE
Improve index page

### PR DESCRIPTION
**Pull request by TIMMLOPK**
1.  Replace Chakra UI Image with next/image

> Chakra Image don’t have the optimization of next js.

2. Remove chakra Icon props

> It is useless props

### CODE REVIEW

**Review by Harry-Hopkinson — APPROVED** [review]
Looks Good

**Comment by TIMMLOPK** [comment]
All thing are working , Should we merge this ? @craftzdog

**Review by craftzdog — COMMENTED** [review]
[968bb45](https://github.com/craftzdog/craftzdog-homepage/pull/27/commits/968bb450971576afe5bf3c86b0c0ee6e4f8e4b34)
Looks better, but why do you think it is necessary to use `next/image` for such a small image?

**Review by craftzdog — APPROVED** [review]
[bf1dd15](https://github.com/craftzdog/craftzdog-homepage/pull/27/commits/bf1dd15d0ff9fb5b364c9fbaade6ac7e5cb07e47) This commit looks great

**Comment by TIMMLOPK** [comment]
> [968bb45](https://github.com/craftzdog/craftzdog-homepage/pull/27/commits/968bb450971576afe5bf3c86b0c0ee6e4f8e4b34)
> Looks better, but why do you think it is necessary to use `next/image` for such a small image?

The image location at index page ,it maybe improve first contentful paint time so it will be batter than and I saw the next/image is used on other page

**Comment by TIMMLOPK** [comment]
> > [968bb45](https://github.com/craftzdog/craftzdog-homepage/pull/27/commits/968bb450971576afe5bf3c86b0c0ee6e4f8e4b34)
> > Looks better, but why do you think it is necessary to use `next/image` for such a small image?
> 
> The image location at index page ,it maybe improve first contentful paint time so it will be batter than and I saw the next/image is used on other page

Currently Next JS 12.1,it improve next/image again 

> The built-in Image Optimization API has been updated to support the same pattern as [ISR pages](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration), where images are served stale and revalidate in the background (also known as stale-while-revalidate).This is a significant performance improvement for optimizing images. See [Image Caching Behavior](https://nextjs.org/docs/api-reference/next/image#caching-behavior) to learn more.

File: pages/index.js [hunk 1]
```diff
@@ -4,13 +4,12 @@ import {
...
   Heading,
   Box,
-  Image,
   SimpleGrid,
   Button,
   List,
   ListItem,
-  Icon,
-  useColorModeValue
+  useColorModeValue,
```
Comment (craftzdog) [inline pages/index.js:11]: Can you remove it?

Comment (TIMMLOPK) [reply pages/index.js:11]: You mean remove Icon?

File: pages/index.js [hunk 2]
```diff
@@ -48,16 +52,24 @@ const Home = () => (
           ml={{ md: 6 }}
           textAlign="center"
         >
-          <Image
+          <Box
```
Comment (craftzdog) [inline pages/index.js:55]: I don't know but does it have to be wrapped with `Box`?
Can't `ProfileImage` have those props just like chakra's `Image` component due to `next/image`?

File: pages/index.js (hunk 1 above)
Comment (craftzdog) [reply pages/index.js:11]: No, seems `useColorModeValue` isn't used.

Comment (craftzdog) [reply pages/index.js:11]: ah sorry, I was wrong

File: pages/index.js (hunk 2 above)
Comment (TIMMLOPK) [reply pages/index.js:55]: Yes,it is picture border

**Comment by TIMMLOPK** [comment]
Just like I said ,next js 12.1 have a new feature for image.I think updating the next js to 12.1 is great 😍

**Comment by craftzdog** [comment]
Okay, thanks!

**Pull request merged**